searx/webapp.py

```diff
@@ -898,19 +906,36 @@ def __get_translated_errors(unresponsive_engines: Iterable[UnresponsiveEngine]):
 @app.route('/about', methods=['GET'])
 def about():
     """Redirect to about page"""
-    return redirect(url_for('help_page', pagename='about'))
+    locale = request.preferences.get_value('locale')
+    return redirect(url_for('info', pagename='about', locale=locale))
 
 
-@app.route('/help/en/<pagename>', methods=['GET'])
-def help_page(pagename):
-    """Render help page"""
-    page = user_help.PAGES.get(pagename)
+_INFO_PAGES = infopage.InfoPageSet()
 
+
+@app.route('/info/<locale>/<pagename>', methods=['GET'])
+def info(pagename, locale):
+    """Render page of online user documentation"""
+
+    page = _INFO_PAGES.get_page(pagename, locale)
     if page is None:
         flask.abort(404)
 
+    def all_pages():
```

Comment:
Iterator over all the page of the TOC including:
* pagename
* page
* locale

If the locale is `de` and `de/about.md` exists (and no other translated page), this function is going to yield:
* `search_syntax`, `PageInfo(....)`, `en`
* `about`, `PageInfo(....)`, `de`